src/solve/Solver.hpp

```diff
@@ -153,10 +153,10 @@ class Solver
     }
 
     // Main solver function.
-    // Input: an array of 512 x1 candidates.
```

Comment:
I'm pretty sure 512 is wrong here. But I'm having second thoughts about 256 as well. is it 128?